rsync_to_rdisc.py

```diff
@@ -334,24 +335,35 @@ def upload_exomedepth_vcf(run, run_folder):
 
 
 if __name__ == "__main__":
-
-    """If daemon is running exit, else create transfer.running file and continue."""
+    # If daemon is running exit, else create transfer.running file and continue.
     run_file = check_daemon_running(settings.wkdir)
+    remove_run_file = True
 
-    """Check if mount to BGarray intact."""
-    check_mount(settings.bgarray, run_file)
-
-    """Make set of transferred_runs.txt file, or create transferred_runs.txt if not present."""
+    # Make set of transferred_runs.txt file, or create transferred_runs.txt if not present.
     transferred_set = get_transferred_runs(settings.wkdir)
 
-    """Get folders to be transfer from HPC."""
+    # Connect to hpc
     client, hpc_server = connect_to_remote_server(settings.host_keys, settings.server, settings.user, run_file)
-    to_be_transferred = get_folders_remote_server(client, settings.folder_dic, run_file, transferred_set)
 
-    """Rsync folders from HPC to bgarray."""
-    remove_run_file = rsync_server_remote(hpc_server, client, to_be_transferred, run_file)
+    # Run rsync commands for each mount point.
+    for mount_name in settings.transfer_settings:
+        mount_path = settings.transfer_settings[mount_name]['mount_path']
+        # Check if mount is available and continue
+        if is_mount_available(mount_name, mount_path, run_file):
```

Comment:
Is it relevant to check if a mount is available, when nothing will be transferred and mount will not be used?

Reply:
Yes I think we always want to know if one of our mounts are lost. To make sure we can fix it asap :).